### ISSUE TITLE
Bug: Multicurrency payments not deducting from account

### ISSUE THREAD

**Issue by MartinFulgueiras** [Dolibarr/dolibarr#7513]
# bug

## Environment
- **Version**: 6.0.1
- **OS**: Linux
- **Web server**: Apache
- **PHP**: cgi-fcgi 5.5.38
- **Database**: MySQL or MariaDB 5.6.37
- **URL**: /crmd/compta/bank/ligne.php?rowid=1820

## Report

Hi, There is this issue in multi currency payments. I had an invoice in PEN (local currency). I paid it from a USD account. The payment was done correctly through the pay invoice button and the invoice was left as paid in local currency. All good here and you can see this in the following image. 
![screenshot-2017-9-27 dolibarr](https://user-images.githubusercontent.com/32270312/30924299-f9145c22-a373-11e7-9bf6-e9604a5f9922.png)

But because there was a multi currency operation involved, the deduction in the USD account was 0.00 as you can see here 
![screenshot-2017-9-27 registro bancario](https://user-images.githubusercontent.com/32270312/30924704-23809bc8-a375-11e7-938c-e3ac2a6bf3c3.png)

To solve it I have to go to the bank account and manually input the amount in the foreign currency. 

An idea could be to add an exchange rate box beside the amount to pay (at the payment) so that the system can do the math when managing multicurrency. Also input both amounts, local and foreign at the payment. 

Thanks for the great work! Hope this helps

**Comment by AlainRnet** [comment]
Hi.
I found your issue and I comment because I encounter a similar problem.
There seems to be a problem with invoices created before, except for error, switching to v.6 and adding a second field for the amounts.
I have two down payment invoices in CHF and a final invoice also in CHF but created after the Dolibarr update from v.5 to v.6
In the total, the deduction is well taken into account, but not in the PDF. In the invoice lines, Total HT does include the amount of the down payment, whereas Total HT (original currency) is 0. And it is apparently this column that is taken into account to generate the PDF, because the total does not deduct the down payments.

I spotted a similar problem yesterday or the day before yesterday when the total amount was 0 in the PDF, so I added the amount in the `multicurrency_total_ht` and` multicurrency_total_ttc` fields of the llx_facture table.

I just checked my llx_facturedet table and the lines of the two installments have a `total` and a` total_ttc` but 0 in `multicurrency_total_ht` and` multicurrency_total_ttc`.
